### ISSUE TITLE
Role Management Page has Incorrect Heading Levels

### ISSUE THREAD

**Issue by Scott-James-Hurley** [UserOfficeProject/user-office-project-issue-tracker#1675]
On the role management page, the "Role Management" header is level 4 but the "Existing Roles" header is level 2 which is lower and smaller. A heading's size should shrink and its position should be lower on the page as its level increases. For example, a level 2 heading should be large and higher up on the page than a level 3 heading.
